src/main/java/se/kth/meta/db/Dbao.java

```diff
@@ -196,11 +191,13 @@ public void deleteTable(Tables table) throws DatabaseException {
 
     try {
       Tables t = this.getTable(table.getId());
-      t.setForceDelete(table.forceDelete());
-      if (!t.getFields().isEmpty() && !t.forceDelete()) {
-        throw new DatabaseException("Table '" + t.getName() + "' has fields "
-                + "associated to it");
-      }
+
+//    NEEDS TO BE REEMPLOYED
```

Comment:
The front end is not aware of the controlled exception thrown here. The exception indicates that the table or field going to be deleted contains metadata